wireapp/wire-ios-mono
# fix: message database migration 116 to 117 - WPB-10032

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-10032" title="WPB-10032" target="_blank"><img alt="Task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10818?size=medium" />WPB-10032</a>  iOS: fix message database migration 116 to 117
  </td></table>
  <br />
 











### Issue

- The step from 116 was not migrating to 117. Now it does.
- Simplified namings of migration versions


### Testing

version 117 just adds a mls metadata field, so it's not easy to test the migration with the app itself.
developers can do:

- install an app version with 116 as last model (or modify `zmessaging.xcdatamodel` current version)
- run the app
- set break points in the migration `CoreDataMessagingMigrator`
- install the app from this branch
- see that the migration is performed correctly

```console
[setup] start migration of core data messaging store!
messaging core data store migration step v116 to v117
finish migrate store for v116
finish migration step for v116
replace store
replace store finished
destroy last store
[setup] finished migration of core data messaging store!
time spent in migration only: 0.9810850620269775
[setup] load core data stores!
```

---

### Checklist
